common/app/views/fragments/inlineJSBlocking.scala.html

```diff
@@ -59,6 +58,11 @@
     }
 
 
+    // ************* ANALYTICS *************
+
+    // Ophan pageview ID and browser ID are needed by Google Analytics, which runs sync at the bottom of the page
+    @InlineJs(ophanConfig().body, "ophanConfig.js")
```

Comment:
@sndrs Can you see any problem with moving the Ophan pageview ID generation from the `utils/config` AMD module to the inline blocking JS section like this? The reason for doing so is to make it available at page load time so the GA code can read it.